flink-runtime-web/README.md

```diff
@@ -102,6 +102,8 @@ re-compilation:
 
 ### CodeStyle & Lint
 
+> Tips: Before committing, make sure you've run npm run lint-staged and that no errors are reported.
```

Comment:
To enhance clarity and readability, it might be beneficial to format the command within the text. This helps to differentiate the command from the rest of the sentence and indicates that it is a piece of code that should be executed. Here's how it could look:

Before committing, make sure you've run `npm run lint-staged` and that no errors are reported.

Reply:
I'll fix this during merging.